datascience/tables.py

```diff
@@ -2260,6 +2331,16 @@ def plot(self, column_for_xticks=None, select=None, overlay=True, width=6, heigh
         >>> table.plot('days', 'price') # doctest: +SKIP
         <line graph with days as x-axis and line for price>
         """
+        global _INTERACTIVE_PLOTS
+        if _INTERACTIVE_PLOTS:
+            return self.iplot(column_for_xticks, select, overlay, width, height, **vargs)
+
+        if width is None:
```

Comment:
No need for this. The _visualize method already has default params for this, unless I'm missing something...

Reply:
I think all of these are necessary... `Table#_visualize` specifies the defaults of the width and height as default values for kwargs but since it is called with these kwargs set to the values in the function (i.e. `self._visualize(..., width=width, height=height, ...)`) these get overwritten with `None`. Will put these into a class attribute so that we don't hardcode the same values multiple times but I think these statements are necessary.

Reply:
ahh you're right, I was thinking of something else. good catch!